step-templates/Jenkins-Queue-Job

```diff
@@ -0,0 +1,71 @@
+{
+  "Id": "ccb7ad4c-a19e-426f-822e-cd0e0243bda3",
+  "Name": "Queue Jenkins Job",
+  "Description": "Trigger a job in Jenkins",
+  "ActionType": "Octopus.Script",
+  "Version": 1,
+  "CommunityActionTemplateId": null,
+  "Properties": {
+    "Octopus.Action.Script.Syntax": "PowerShell",
+    "Octopus.Action.Script.ScriptSource": "Inline",
+    "Octopus.Action.RunOnServer": "false",
+    "Octopus.Action.Script.ScriptBody": "$JenkinsServer = \"http://jenkins:8080/\"\r\n$jobURL = $JenkinsServer + $OctopusParameters['JobUrl']\r\n$failBuild = $OctopusParameters['FailBuild']\r\n$jobTimeout = $OctopusParameters['JobTimeout']\r\n$buildParam = $OctopusParameters['BuildParam']\r\n$buildResult = \"/lastBuild/api/json?tree=result,number,building\"\r\n\r\nWrite-Host \"job url: \" $jobURL$buildParam \r\n$response = Invoke-WebRequest -Uri $jobURL$buildParam  -Method Post\r\n\r\nif ($failBuild -eq \"True\")\r\n{\r\n   $isBuilding=\"True\"\r\n   $i = 0\r\n   Write-Host \"Estimate Job Duration: \" $jobTimeout\r\n   while ($isBuilding -eq \"True\" -and $i -lt $jobTimeout)\r\n   {       \r\n       $i +=20\r\n       Write-Host \"waiting $i secs for build to complete\"\r\n       Start-Sleep -s 20\r\n       $retyJobStatus = Invoke-RestMethod -Uri $jobURL$buildResult -Method Get\r\n       $isBuilding=$retyJobStatus[0].building\r\n       $result = $retyJobStatus[0].result\r\n       $buildNumber = $retyJobStatus[0].number\r\n       Write-Host \"Retry Job Status: \" $result \" BuildNumber: \"  $buildNumber  \" IsBuilding: \"  $isBuilding \r\n   }\r\n   \r\n   if ($result -ne \"SUCCESS\")\r\n   {\r\n       echo \"BUILD FAILURE: build is unsuccessful or status could not be obtained.\"\r\n       exit 1\r\n   }\r\n}",
```

Comment:
`$retyJobStatus = Invoke-RestMethod -Uri $jobURL$buildResult -Method Get`
Should be `$retryJobStatus = ...`